target/scripts/startup/setup.d/security/misc.sh

```diff
@@ -81,11 +81,6 @@ function __setup__security__spamassassin() {
     # shellcheck disable=SC2016
     sed -i -r 's|^\$sa_kill_level_deflt (.*);|\$sa_kill_level_deflt = '"${SA_KILL}"';|g' /etc/amavis/conf.d/20-debian_defaults
 
-    # fix cron.daily for spamassassin
-    sed -i \
-      's|invoke-rc.d spamassassin reload|/etc/init\.d/spamassassin reload|g' \
-      /etc/cron.daily/spamassassin
-
```

Comment:
Why was this fix removed?

If I understand correctly, the logic 

```
# Reload
if which invoke-rc.d >/dev/null 2>&1; then
    invoke-rc.d --quiet spamd status > /dev/null && \
      invoke-rc.d spamd reload > /dev/null
else
    /etc/init.d/spamd reload > /dev/null
fi
if [ -d /etc/spamassassin/sa-update-hooks.d ]; then
    run-parts --lsbsysinit /etc/spamassassin/sa-update-hooks.d
fi
```

moved to `/usr/sbin/spamassassin-maint` and should be fixed there now?

Suggestion:

```bash
# fix cron.daily for spamassassin
sed -i \
  's|invoke-rc.d spamd reload|/etc/init.d/spamd reload|g' \
  /usr/sbin/spamassassin-maint
```

Reply:
Good catch!

Reply:
See my [earlier review feedback](https://github.com/docker-mailserver/docker-mailserver/pull/3403#pullrequestreview-1596689953), specifically the last part of this screenshot:

![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/a9fafda0-cfcb-42ff-a40c-2a916148071b)


https://github.com/docker-mailserver/docker-mailserver/blob/c8a0bfd36195281b629916d64edf0a2c1fbd89d1/target/scripts/startup/setup.d/security/misc.sh#L82-L85

Verbose review comment related to those screenshot suggestions [can be found here](https://github.com/docker-mailserver/docker-mailserver/pull/3403#discussion_r1306282387). Here is the relevant context from that comment:

![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/e04092ff-b392-4c88-8d0c-5a31328010dd)